system-test/spanner.ts

```diff
@@ -252,37 +302,60 @@ describe('Spanner', () => {
           },
         ];
 
+        let table = googleSqlTable;
+        let database = DATABASE;
+        let query: ExecuteSqlRequest = {
+          sql: `SELECT * FROM \`${table.name}\` WHERE Key = @a OR KEY = @b`,
+          params: {
+            a: data[0].Key,
+            b: data[1].Key,
+          },
+        };
+        if (dialect === Spanner.POSTGRESQL) {
+          table = postgreSqlTable;
+          database = PG_DATABASE;
+          query = {
+            sql: `SELECT * FROM ${table.name.toLowerCase()} WHERE "Key" = $1 OR "Key" = $2`,
```

Comment:
nit: I don't think we need the `.toLowerCase()` here. The table name will be folded to lowercase automatically by the backend.